extensions/resource-deployment/src/services/resourceTypeService.ts

```diff
@@ -253,12 +250,14 @@ export class ResourceTypeService implements IResourceTypeService {
 
 	public startDeployment(resourceType: ResourceType, provider: DeploymentProvider): void {
 		const self = this;
-		if (instanceOfWizardDeploymentProvider(provider)) {
+		if (
+			instanceOfWizardDeploymentProvider(provider) ||
```

Comment:
All of these are ultimately ResourceTypeWizards does that present an opportunity to simplify this so that when and new resoruceTypeWizard gets added we do not need to add another || check here?

Reply:
Like in the worst case this could be the catchall else if it is not an instanceOfDialogDeploymentProvider or commandDeploymentProvider